mozci/push.py

```diff
@@ -1132,27 +1131,46 @@ def __make_group_summary_objects(pushes):
     """
     # Obtain all the task.id values contained in the pushes. It will be used in
     # the `where` query against ActiveData.
-    task_ids = [task.id for push in pushes for task in push.tasks]
-
-    # Extract the results of the above query.
-    results = run_query("group_durations", Namespace(task_id=task_ids))["data"]
+    results = []
+    revs = [p.rev for p in pushes]
+    start = 0
+    revcount = 0
+    try:
```

Comment:
The try/catch should be inside the loop, otherwise the first query with missing data will stop all other queries.